pkg/issuer/acme/http/ingress.go

```diff
@@ -33,6 +33,14 @@ import (
 	logf "github.com/jetstack/cert-manager/pkg/logs"
 )
 
+const (
+	// annotationIngressClass is the well-known annotation key
+	// for specifying ingress classes. It is currently not specified
+	// in the networking/v1 package, so it is duplicated here
+	// to avoid an extra import of networking/v1beta1
```

Comment:
I apologize in advance for my useless suggestion 😇
```suggestion
	// to avoid an extra import of networking/v1beta1.
```